lib/deliver/submit_for_review.rb

```diff
@@ -32,38 +32,49 @@ def select_build(options)
       Helper.log.info "Selecting the latest build..."
       app = options[:app]
       v = app.edit_version
+      build = wait_for_build(app)
 
-      start = Time.now
+      Helper.log.info "Selecting build #{build.train_version} (#{build.build_version})..."
 
-      build = nil
-      v.candidate_builds.each do |b|
-        if !build or b.upload_date > build.upload_date
-          build = b
-        end
-      end
+      v.select_build(build)
+      v.save!
 
-      unless build
-        Helper.log.fatal v.candidate_builds
-        raise "Could not find build to select".red
-      end
+      Helper.log.info "Successfully selected build".green
+    end
+
+    def wait_for_build(app)
+      raise "Could not find app with app identifier #{WatchBuild.config[:app_identifier]}".red unless app
+
+      start = Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.